components/esp_lvgl_port/README.md

```diff
@@ -99,6 +99,50 @@ Add touch input to the LVGL. It can be called more times for adding more touch i
 
 **Note:** The inputs added in this function are not removed in `lvgl_port_deinit`. They must be removed by `lvgl_port_remove_touch` before deinitialization. Otherwise, there can be memory leaks!
 
+### Add buttons input
+
+Add buttons input to the LVGL. It can be called more times for adding more buttons inputs for different displays. 
```

Comment:
Maybe now we can add a sentence here that this feature is available if `espressif/button` component is included.